storage/mysql/rootStore.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS Roots(
+  Id BINARY(32),
+  DER MEDIUMBLOB,
+  PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS RootSets(
+  RootSetID Binary(32),
+  RootID Binary(32),
+  PRIMARY KEY(RootSetID, RootID)
+);
+
+CREATE INDEX RootSetIDIndex ON RootSets(RootSetID);
```

Comment:
There's already an index on RootSetID as a result of it being the first column in the primary key.

Reply:
Removed.